packages/core/src/shared/utilities/pollingSet.ts

```diff
@@ -44,9 +44,14 @@ export class PollingSet<T> extends Set<T> {
             this.clearTimer()
         }
     }
-
+    // TODO(hkobew): Overwrite the add method instead of adding seperate method. If we add item to set, timer should always start.
```

Comment:
plan to do as quick follow up, since this involves touching some other files unrelated to process work.

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/6396